src/components/detector.ts

```diff
@@ -121,14 +147,25 @@ export default class Detector extends EventEmitter implements IDetector {
         'IQ Detector teardown by stopping detection generation on previewstream, unsubscribing to events and unregistering listeners',
         'IQ Detector'
       );
-      // Send `detector/stop` only if `detector/start` was called previously
-      await this._deviceManager.transport.write('detector/stop');
+      // Send `[people_count/detector]/stop` only if `[people_count/detector]/start` was called previously
+      if (this._usePeopleCount) {
```

Comment:
If you move this outside the if you don't have to repeat, it since you can always stop people_count if it is there